docs/config.md

```diff
@@ -1456,13 +1456,14 @@ signature details.
 
 ### Default colocation
 
-When creating a git-backed Jujutsu repository, you can disable colocation which
-places the `.git` directory next to the `.jj` directory. Colocation allows some
-amount of two-way interoperability, but it can perform worse in large repos.
+When creating a Git-backed Jujutsu repository, they use "colocated" mode by
+default, which places the `.git` directory next to the `.jj` directory.
+Colocation allows some amount of two-way interoperability, but it can perform
+worse in large repos.
 
 The setting `git.colocate` is a boolean option that controls whether or not the
```

Comment:
(Possibly for another PR): Perhaps something like:

"The default can be changed with the `git.colocate` boolean option. It controls .... by default. You can set `git.colocate` to false to default to non-colocated options, and you can override `git.colocate` with the `--colocate` or `--no-colocate` CLI options"

I'm thinking this makes the meaning of "default" slightly clearer, and I think it makes sense to mention the CLI options (or does this overlap with 

Relatedly, there's also https://github.com/jj-vcs/jj/issues/7776 (the CLI docs are also off).

Do you prefer to submit this as is and then have me (or someone else) work on these things later, or to do some or all of these things yourself?